build.gradle

```diff
@@ -119,6 +123,16 @@ subprojects { project ->
     errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
     errorprone 'com.google.errorprone:error_prone_core:2.3.3'
   }
+
+  apply plugin: 'org.jetbrains.dokka'
+  dokka {
```

Comment:
Some other useful ones 
- `Module.md` files
- `externalDocumentationLink` for dependencies

```gradle
externalDocumentationLink {
    url = new URL("https://docs.oracle.com/javase/7/docs/api/")
}
```
- `includeNonPublic = false`
- `outputFormat = 'html'`

Reply:
and for docs referencing other sibling projects as deps: https://github.com/Kotlin/dokka/issues/269#issuecomment-468733943

```gradle
evaluationDependsOn(":okhttp")
afterEvaluate {
  dokka {
    classpath = configurations.compile
    externalDocumentationLink {
      packageListUrl = new URL("file://${project(":okhttp").buildDir}/dokka/okhttp/package-list")
      url = new URL("file://${project(":okhttp").buildDir}/dokka/okhttp/")
    }
  }
}
```